bwctest.sh

```diff
@@ -50,50 +60,27 @@ while getopts ":ha:p:b:c:" arg; do
     esac
 done
 
+[ -z "$BIND_ADDRESS" ] && BIND_ADDRESS="localhost"
+[ -z "$BIND_PORT" ] && BIND_PORT="5601"
+[ -z "$SECURITY_ENABLED" ] && SECURITY_ENABLED="false"
+[ -z "$CREDENTIAL" ] && CREDENTIAL="admin:admin"
 
-if [ -z "$BIND_ADDRESS" ]
-then
-  BIND_ADDRESS="localhost"
-fi
-
-if [ -z "$BIND_PORT" ]
-then
-  BIND_PORT="5601"
-fi
-
-if [ -z "$BUNDLED_OSD" ]
-then
-  BUNDLED_OSD="true"
-fi
-
-if [ -z "$CREDENTIAL" ]
-then
-  CREDENTIAL="admin:admin"
-  USERNAME=`echo $CREDENTIAL | awk -F ':' '{print $1}'`
-  PASSWORD=`echo $CREDENTIAL | awk -F ':' '{print $2}'`
-fi
-
-cwd=$(pwd)
-dir="bwc-tmp"
-if [ -d "$dir" ]; then
-  rm -rf "$dir"
-  echo "bwc-tmp exists and needs to be removed"
-fi
-
-mkdir "$dir"
-git clone https://github.com/opensearch-project/opensearch-dashboards-functional-test "$dir"
-rm -rf "$dir/cypress"
-cp -r cypress "$dir"
-cd "$dir"
-
-npm install
+# If no OpenSearch build was passed then this constructs the version
+if [ -z "$OPENSEARCH" ]; then
+    IFS='/' read -ra SLASH_ARR <<< "$DASHBOARDS"
+    # Expected to be opensearch-x.y.z-platform-arch.tar.gz
+    TARBALL="${SLASH_ARR[12]}"
+    IFS='-' read -ra DASH_ARR <<< "$TARBALL"
+    # Expected to be arch.tar.gz
+    DOTS="${DASH_ARR[4]}"
+    IFS='.' read -ra DOTS_ARR <<< "$DOTS"
+    
+    VERSION="${DASH_ARR[2]}"
+    PLATFORM="${DASH_ARR[3]}"
+    ARCH="${DOTS_ARR[0]}"
 
-if [ $BUNDLED_OSD = "true" ]
-then
-   echo "run security enabled tests"
-   npx cypress run --spec "$cwd/bwc-tmp/cypress/integration/bundled-osd/*.js"
-else
-   npx cypress run --spec "$cwd/bwc-tmp/cypress/integration/osd/*.js"
+    # TODO: Replace '676' with -latest from distributions
```

Comment:
Can we do this now?

Reply:
Not yet, until this one gets completed: https://github.com/opensearch-project/opensearch-build/issues/1492

Reply:
Cool, can we add a reference to that issue?